packages/app/src/server/service/file-uploader/uploader.js

```diff
@@ -12,6 +19,30 @@ class Uploader {
     return !this.configManager.getConfig('crowi', 'app:fileUploadDisabled') && this.isValidUploadSettings();
   }
 
+  /**
+   * Get whether write opration to the storage is permitted
+   * @returns Whether write opration to the storage is permitted
+   */
+  async getIsWritable() {
```

Comment:
```suggestion
  async isWritable() {
```
じゃダメなの？
`getIsUploadble` とか `getIsReadble` とか気持ち悪い…